.forceignore

```diff
@@ -0,0 +1,3 @@
+**/jsconfig.json
```

Comment:
The change to this file is not needed. The same is applicable to the `datasets/dev_org/test_data.db` file, it is not required for this PR.